### PR TITLE
html-forms | Richard Watanabe

### PR DESCRIPTION
**Pull request by Richard-Watanabe**
Screen Shot 2021-06-22 at 12 50 43 PM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <form action="index.html">
+    <h1>Coding Form!</h1>
+    <div>Email</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:9]: This should be the `label` element, not a `div`. This is the text your user would click to try and put focus on the input it pairs with.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <label for="email">
+      <input type="text" name="email" id="email">
+    </label>
+    <div>Why is coding cool?</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:13]: This should be the `label` element, not a `div`. This is the text your user would click to try and put focus on the input it pairs with.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <label for="JavaScript">
+      <input type="radio" name="favorite" value="JavaScript" id="JavaScript">
+      JavaScript
+    </label>
+    <div>Would you like to sign up for more information?</div>
+    <label for="info">
+      <input type="checkbox" name="info" id="info" value="Sign me Up!">
+      Sign me up!
+    </label>
+    <div>Comments</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:40]: This should be the `label` element, not a `div`. This is the text your user would click to try and put focus on the input it pairs with.

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    <label for="comments">
+      <textarea name="comments" id="comments" cols="23" rows="10"></textarea>
+    </label>
+    <div>
+    <label for="submit">
```
Comment (codingcodymiller) [inline html-forms/index.html:45]: There is no `label` element necessary here, since there is no label text to interact with.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start on this! A few notes for you:

**Comment by Richard-Watanabe** [comment]
Tried fixing the labels for the *html-forms exercise^

**Comment by Richard-Watanabe** [comment]
Screen Shot 2021-06-22 at 1 28 36 PM

**Review by codingcodymiller — APPROVED** [review]
Great work!!!

**Pull request merged**